README.md

```diff
@@ -201,7 +201,7 @@ response = chatbot.predict("Tell me about Intel Xeon Scalable Processors.")
 Below is the sample code to use the extended Transformers APIs. See more [examples](https://github.com/intel/neural-speed/tree/main).
 
 #### INT4 Inference (CPU)
-We encourage you to install [NeuralSpeed](https://github.com/intel/neural-speed) to get the latest features (e.g., GGUF support) of LLM low-bit inference on CPUs. You may also want to use v1.3 without NeuralSpeed by following the [document](https://github.com/intel/intel-extension-for-transformers/tree/v1.3/intel_extension_for_transformers/transformers/llm/runtime/neural_speed/README.md)
+We encourage you to install [NeuralSpeed](https://github.com/intel/neural-speed) to get the latest features (e.g., GGUF support) of LLM low-bit inference on CPUs. You may also want to use v1.3 without NeuralSpeed by following the [document](https://github.com/intel/intel-extension-for-transformers/tree/v1.3/intel_extension_for_transformers/llm/runtime/graph/README.md)
```

Comment:
This doc is about Neural Speed. Why does it write `use v1.3 without NeuralSpeed by following this [document]` ?